src/daemon/auth/providers/openid_connect/config.rs

```diff
@@ -34,66 +33,30 @@ pub struct ConfigAuthOpenIDConnect {
 }
 
 fn default_prompt_for_login() -> bool {
-    // On by default for backward compatability. See: https://github.com/NLnetLabs/krill/issues/614
+    // On by default for backward compatability.
+    // See: https://github.com/NLnetLabs/krill/issues/614
     true
 }
 
-#[derive(Clone, Debug, Deserialize)]
-pub struct ConfigAuthOpenIDConnectClaim {
-    pub source: Option<ConfigAuthOpenIDConnectClaimSource>,
-    pub jmespath: Option<String>,
-    pub dest: Option<String>,
-}
-
-#[derive(Clone, Debug)]
-pub enum ConfigAuthOpenIDConnectClaimSource {
-    ConfigFile,
-    IdTokenStandardClaim,
-    IdTokenAdditionalClaim,
-    UserInfoStandardClaim,
-    UserInfoAdditionalClaim,
+fn default_id_claims() -> Vec<TransformationRule> {
```

Comment:
Perhaps make it more explicit in the config/manual that this is the default (and does not need to be manually configured)

Reply:
I added a heading above the text in the manual to make it easier to find.